modules/openapi-generator/src/main/resources/kotlin-client/libraries/jvm-volley/api.mustache

```diff
@@ -0,0 +1,129 @@
+package {{apiPackage}}
+
+import android.content.Context
+import com.android.volley.DefaultRetryPolicy
+import com.android.volley.Request
+import com.android.volley.RequestQueue
+import com.android.volley.Response
+import com.android.volley.toolbox.BaseHttpStack
+import com.android.volley.toolbox.Volley
+import java.util.*;
+import kotlin.coroutines.resume
+import kotlin.coroutines.resumeWithException
+import kotlin.coroutines.suspendCoroutine
+import com.google.gson.reflect.TypeToken
+
+import {{packageName}}.request.IRequestFactory
+import {{packageName}}.request.RequestFactory
+import {{packageName}}.infrastructure.CollectionFormats.*
+
+{{#imports}}import {{import}}
+{{/imports}}
+
+{{#operations}}
+/*
+* If you wish to use a custom http stack with your client you
+* can pass that to the request queue like:
+* Volley.newRequestQueue(context.applicationContext, myCustomHttpStack)
+*/
+class {{classname}} (
+    val context: Context,
+    val requestQueue: Lazy<RequestQueue> = lazy(initializer = {
+        Volley.newRequestQueue(context.applicationContext)
+    }),
+    val requestFactory: IRequestFactory = RequestFactory(),
+    val basePath: String = "{{{basePath}}}",
```

Comment:
@alisters could you please make all of those private?

Reply:
Done